lib/sift/client.rb

```diff
@@ -121,26 +121,32 @@ def user_agent
     #   Overrides the default API path with a different URL.
     #
     # return_score (optional)
-    #   Whether the API response should include a score for this user (the score will
+    #   Whether the API response should include a score for this user. The score will
     #   be calculated using the submitted event.  This feature must be
     #   enabled for your account in order to use it.  Please contact
     #   support@siftscience.com if you are interested in using this feature.
     #
+    # return_action (optional)
+    #   Whether the API response should include an action triggered for this transaction.
+    #
     # == Returns:
     #   In the case of an HTTP error (timeout, broken connection, etc.), this
     #   method returns nil; otherwise, a Response object is returned and captures
     #   the status message and status code. In general, you can ignore the returned
     #   result, though.
     #
-    def track(event, properties = {}, timeout = nil, path = nil, return_score = false, api_key = @api_key)
+    def track(event, properties = {}, timeout = nil, path = nil, return_score = false, return_action = false, api_key = @api_key)
```

Comment:
Please add the return_action argument to the end of the list for maintaining backwards compatibility